nengo_spinnaker/regions/filters.py

```diff
@@ -35,21 +35,22 @@ def make_filter_regions(signals_and_connections, dt, minimise=False,
     filters = list()
     keyspace_routes = list()
 
-    for signal, connections in iteritems(signals_and_connections):
-        for connection in connections:
-            # Make the filter
-            f = FilterRegion.supported_filter_types[type(connection.synapse)].\
-                from_signal_and_connection(signal, connection, width=width)
+    for signal, rps in specs:
```

Comment:
Being pedantic, I think `rps` is overly terse  personally I'd call it's `reception_params`, but even `params` would make it's role clearer